rbkit-lib/cpu/cpumapping.h

```diff
@@ -0,0 +1,28 @@
+#include <QString>
+
+#ifndef MAPPING
+#define MAPPING
+
+namespace RBKit {
```

Comment:
in client we are using `RBKit` [stringutil.h](https://github.com/code-mancers/rbkit-client/blob/master/rbkit-lib/stringutil.h#L10)